barman/backup.py

```diff
@@ -1467,6 +1467,12 @@ def backup_fsync_and_set_sizes(self, backup_info):
                 file_path = os.path.join(dir_path, filename)
                 file_stat = fsync_file(file_path)
                 backup_size += file_stat.st_size
+
+                if backup_info.parent_backup_id is not None:
```

Comment:
This is just a scratch! Most probably incorrect but i wanted to leave it with something so we can think about the best way to achieve the goal to calculate sizes and deduplicates sizes.